doc/14_data_frames.md

```diff
@@ -23,38 +25,64 @@ Those followed with a default of N/A are required, all others are optional.
-can be passed through as DataFrame options as well. When using the `read` command below these
-options should appear exactly the same as when set in the SparkConf.
+can be passed through as DataSet options as well. When using the `read` command below these
+options should appear exactly the same as when set in the SparkConf. See 
+[Config Helpers](#example_using_typesafe_parameter_configuration_options) for
+typed helpers for setting these options.
 
 #### Setting Cluster and Keyspace Level Options
 The connector also provides a way to describe the options which should be applied to all
-DataFrames within a cluster or within a keyspace. When a property has been specified at the
+DataSets within a cluster or within a keyspace. When a property has been specified at the
 table level it will override the default keyspace or cluster property.
 
-To add these properties add keys to your `SparkConf` in the format
+To add these properties add keys to your `SparkConf` use the helpers explained 
+ in the next section or by manually entering them in the format
 
     clusterName:keyspaceName/propertyName.
+    
+#### Example Using TypeSafe Parameter Configuration Options
+There are also some helper method which simplifies setting Spark Cassandra 
+Connector related parameters. This makes it easier to set parameters without
+remembering the above syntax:
+```scala
+import org.apache.spark.sql.cassandra._
+
+import com.datastax.spark.connector.cql.CassandraConnectorConf
+import com.datastax.spark.connector.rdd.ReadConf
+
+// set params for all clusters and keyspaces
+spark.setCassandraConf(CassandraConnectorConf.KeepAliveMillisParam.option(10000))
+
+// set params for the particular cluster
+spark.setCassandraConf("Cluster1", CassandraConnectorConf.ConnectionHostParam.option("127.0.0.1") ++ CassandraConnectorConf.ConnectionPortParam.option(12345))
+spark.setCassandraConf("Cluster2", CassandraConnectorConf.ConnectionHostParam.option("127.0.0.2"))
+
+// set params for the particular keyspace 
+spark.setCassandraConf("Cluster1", "ks1", ReadConf.SplitSizeInMBParam.option(128))
+spark.setCassandraConf("Cluster1", "ks2", ReadConf.SplitSizeInMBParam.option(64))
+spark.setCassandraConf("Cluster2", "ks3", ReadConf.SplitSizeInMBParam.option(80))
+```
 
 #### Example Changing Cluster/Keyspace Level Properties
 ```scala
-sqlContext.setConf("ClusterOne/spark.cassandra.input.split.size_in_mb", "32")
-sqlContext.setConf("default:test/spark.cassandra.input.split.size_in_mb", "128")
+spark.setCassandraConf("ClusterOne", "ks1", ReadConf.SplitSizeInMBParam.option(32))
+spark.setCassandraConf("default", "test", ReadConf.SplitSizeInMBParam.option(128))
 ...
-val df = sqlContext
+val df = spark
   .read
   .format("org.apache.spark.sql.cassandra")
   .options(Map( "table" -> "words", "keyspace" -> "test"))
-  .load() // This DataFrame will use a spark.cassandra.input.size of 32
+  .load() // This DataSet will use a spark.cassandra.input.size of 32
```

Comment:
default value is 128. This should be 128 and `"cluster" -> "ClusterOne"` should be 32